docs/experiments-trec2018.md

```diff
@@ -132,6 +178,30 @@ target/appassembler/bin/SearchCollection -index lucene-index.wash18.pos+docvecto
 target/appassembler/bin/SearchCollection -index lucene-index.wash18.pos+docvectors+rawdocs -topicreader Trec -topics 2018-test-topics.txt -ql -hits 10000 -runtag anserini_qlax17 -axiom -axiom.deterministic -rerankCutoff 0 -axiom.index lucene-index.core17.pos+docvectors+rawdocs -output wash18_ql_ax17.txt
```

Comment:
Should we make all parameters explicit on the command line - e.g. b and k_1 for bm25, etc.
So that if we change defaults later (e.g., for RM3), it'll be easier to reconstruct?

Reply:
I think your suggestion applies more to the general regression test, but not the runbook of TREC?